src/lib/fs.js

```diff
@@ -40,7 +39,7 @@ const rmdirRecursiveAsync = async (path) => {
   if (gte(NODE_VERSION, '14.14.0')) {
     return await rm(path, { force: true, recursive: true })
   }
-  await del(path, { force: true })
+  await rmdir(path, { recursive: true })
```

Comment:
This is breaking the behavior. It does not support the force argument as this is only supported in `rm` which is only avialable from `14.14.0`

Which means if it would be used to delete a file it would fail:

https://nodejs.org/api/fs.html#fspromisesrmdirpath-options

>  Using fsPromises.rmdir() on a file (not a directory) results in the promise being rejected with an ENOENT error on Windows and an ENOTDIR error on POSIX.
>
> To get a behavior similar to the rm -rf Unix command, use fsPromises.rm() with options { recursive: true, force: true }.

As our test coverage is sadly to low I would not do this change as we cannot predict the side effects. I would try to update the minimum node version as fast as possible to `14.14.0` to use `rm`.

cc @erezrokah @ehmicky

Reply:
It appears that `fs.promises.rmdir()` [does not throw `ENOENT`](https://github.com/netlify/cli/pull/4017#discussion_r783122286) on missing directories in Node <14.14.0, but I agree with @lukasholzer this might be safer to wait until we drop Node 12. :+1: